docs/userguides/config.md

```diff
@@ -1,160 +1,315 @@
-# Configure Ape
+# Developing Projects with Ape
```

Comment:
I am confused, why are we changing `config.md` so much? This just looks like the same as `project.md`?
Why is this getting deleted / duplicated??

Reply:
Revert back to OG